src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -460,6 +619,193 @@ subroutine general_read_fv3_regional(this,fv3_filenameginput,g_ps,g_u,g_v,g_tv,g
 
   return       
   end subroutine general_read_fv3_regional
+
+  subroutine general_read_fv3_regional_dirZDA(this,fv3_filenameginput,g_ps,g_u,g_v,g_tv,g_rh,g_oz,g_ql,g_qi,g_qr,g_qs,g_qg,g_qnr,g_w)
```

Comment:
See my comments on changing this subroutine to only read "g_ql,g_qi,g_qr,g_qs,g_qg,g_qnr,g_w".